hortimon/mothership/tickscripts/generic_mean_alert.tick

```diff
@@ -0,0 +1,35 @@
+// which measurement we should consume from, eg 'temperature' or 'humidity'
+var measurement string
+// Optional where filter
+var where_filter = lambda: TRUE
+// Optional list of group by dimensions
+var groups = [*]
+// time window
+var window = 1m
+// Which field to process
+var field = 'value'
+// Warning criteria, has access to 'mean' field
+var warn lambda
+// Critical criteria, has access to 'mean' field
+var crit lambda
+// Critical reset criteria, has access to 'mean' field
+var critReset lambda
+
+stream
```

Comment:
generic alerting template

https://docs.influxdata.com/kapacitor/v1.4/working/template_tasks/

we define variables allowing for easy extension and creating of new similar alerting logic.
the json files in this same directory provide concrete values for these variables and allow us to create new kapacitor alerts using the cli tool

```
docker-compose run kapacitor-cli
$ kapacitor help
```